plugins/org.locationtech.udig.catalog.tests/src/org/locationtech/udig/catalog/tests/internal/CatalogImplTest.java

```diff
@@ -57,4 +64,11 @@ public void testUrlEquals() throws Exception {
     public void testAquire() {
         
     }
+    
+    @Test
+    public void testServiceComparison() throws Exception {
```

Comment:
test acquires a service just by giving the url. The two test services match that request an the better one should be chosen.